wgpu-core/src/device/mod.rs

```diff
@@ -2774,16 +2774,11 @@ impl<A: HalApi> Device<A> {
                             },
                         )?;
                     }
-                    Some(&None) => {
-                        return Err(
-                            pipeline::CreateRenderPipelineError::InvalidFragmentOutputLocation(*i),
-                        );
-                    }
                     _ => {
-                        return Err(pipeline::CreateRenderPipelineError::ColorState(
-                            *i as u8,
-                            pipeline::ColorStateError::Missing,
-                        ));
+                        log::info!(
+                            "The fragment shader output @location({}) values are ignored",
```

Comment:
Could we drop in shader ID/label in here, do we have it available?

Reply:
ShaderModuleId output is like `(0, 1, Metal)`, which is not helpful. entry_point is available but shader label is not.

Reply:
Good 'nuff